core-web/libs/dotcms-scss/angular/dotcms-theme/components/form/_treeselect.scss

```diff
@@ -0,0 +1,98 @@
+@use "variables" as *;
+@import "common";
+
+.p-treeselect {
+    @extend #form-field-base;
+    padding: 0;
+    transition: background-color 0.15s, border-color 0.15s, box-shadow 0.15s;
+    &.p-disabled {
+        @extend #form-field-disabled;
+
+        .p-treeselect-trigger {
+            color: $color-palette-gray-500;
+        }
+    }
+}
+
+.p-treeselect:not(.p-disabled):hover {
+    @extend #form-field-hover;
+}
+
+.p-treeselect:not(.p-disabled).p-focus {
+    @extend #form-field-focus;
+    outline-offset: 0;
+    box-shadow: $shadow-xs;
+}
+
+.p-treeselect .p-treeselect-label {
+    padding: $spacing-1 $spacing-2;
+    transition: background-color 0.15s, border-color 0.15s, box-shadow 0.15s;
+}
+
+.p-treeselect .p-treeselect-label.p-placeholder {
+    color: $color-palette-gray-500;
+}
+
+.p-treeselect.p-treeselect-chip .p-treeselect-token {
+    padding: 0.375rem $spacing-1;
+    margin-right: $spacing-1;
+    background: $color-palette-primary-200;
+    border-radius: $border-radius-sm;
+    color: $color-palette-primary;
+    font-size: $font-size-sm;
+}
```

Comment:
We have styles for chips on: [_chips.scss](https://github.com/dotCMS/core/blob/a2b4e20d1aa71bc0b7a0ec5552cf5fa03686f679/core-web/libs/dotcms-scss/angular/dotcms-theme/components/form/_chips.scss)

Should we update those too?

Reply:
@zJaaal Great, I created a new common to merge these styles and reuse those between chips and TreeSelect.